Assets/_Scripts/Canvases/Menus/Rebind/RebindUI.cs

```diff
@@ -74,12 +81,17 @@ private void Init(NetworkObject player)
         private void InitOnGameStart(GameState newState)
         {
             if (newState == GameState.Running)
+            {  
+                InitWithSpawnedPlayers();
+            }
+        }
+
+        private void InitWithSpawnedPlayers()
+        {
+            GameObject[] players = GameObject.FindGameObjectsWithTag(Tags.PLAYER);
```

Comment:
Can we use the player manager to get the list of players?

Reply:
Yes, mais vu que c'était déjà là, je voulais faire le moins de changement possible (j'ai juste copié le code qu'on avait déjà dans une nouvelle méthode).

À voir si on veut faire un cleanup de code à un moment donné